govcd/catalog_test.go

```diff
@@ -963,3 +963,31 @@ func (vcd *TestVCD) Test_CatalogQueryMediaList(check *C) {
 	// Check that media name is what it should be
 	check.Assert(medias[0].Name, Equals, vcd.config.Media.Media)
 }
+
+// Tests System function UploadMediaImage by checking if provided UDF type standard iso file uploaded.
```

Comment:
> if provided UDF type standard iso file uploaded.

This part of the sentence doesn't make sense. Can you fix it on the way in some of your open PRs?

Reply:
UDF type of ISO file uploaded?

Reply:
We agreed on:

> // Tests System function UploadMediaImage by using provided ISO file of UDF type.

While in commit:

>  // Tests System function UploadMediaImage by checking if provided ISO file of UDF type.

Reply:
@mikeletux updated his PR.